src/routes/supported-features-route.php

```diff
@@ -24,9 +26,7 @@ class Supported_Features_Route implements Route_Interface {
 	 * @return array
 	 */
 	public static function get_conditionals() {
```

Comment:
Suggestion to remove the whole get_conditionals method here as this will be inherited from the 'No_Conditionals' trait

Reply:
Thanks! I missed that one. I've removed it.